xdsl/dialects/riscv.py

```diff
@@ -157,13 +164,87 @@ def print_parameter(self, printer: Printer) -> None:
 
 
 class RISCVOp(Operation, ABC):
-    pass
+    @abstractmethod
+    def assembly_line(self) -> str | None:
+        raise NotImplementedError()
+
+
+_AssemblyInstructionArg: TypeAlias = (
+    AnyIntegerAttr | LabelAttr | SSAValue | RegisterType | str | None
+)
+
+
+class RISCVInstruction(RISCVOp):
+    comment: OptOpAttr[StringAttr]
```

Comment:
Can you add a little docstring here that differentiates the RISCVOp from the RISCVInstruction?